app/gist/controller.js

```diff
@@ -219,6 +291,28 @@ export default Ember.Controller.extend({
       }
     },
 
+    removeColumn (col) {
+      let numColumns = this.get('realNumColumns');
```

Comment:
I'm curious, it seems like rather then making this controller have array like qualities. Having the columns represented off-object would be clean. Something like:

```js
controller.columns === [
  col1,
  col2,
  col3
]
```

```js
columns: computed('dataColumnsNeed', function() {
  return ensureColums(this.get('dataColumnsNeed'))
});
```

the return columns object itself doesn't need to be an array, but may prove to be nice if it ways.

This would also make it easy to support N columns, if we are so inclined, but also begins to move to a pull based system over the imperative model.